feature-libs/quote/occ/adapters/occ-quote.adapter.ts

```diff
@@ -232,6 +232,29 @@ export class OccQuoteAdapter implements QuoteAdapter {
     });
   }
 
+  downloadAttachment(
+    userId: string,
+    quoteCode: string,
+    attachmentId: string
+  ): Observable<Blob> {
+    return this.withErrorHandling(
+      this.httpClient.get<Blob>(
+        this.downloadAttachmentEndpoint(userId, quoteCode, attachmentId),
+        { responseType: 'blob' as 'json' }
+      )
+    );
+  }
+
+  protected downloadAttachmentEndpoint(
```

Comment:
MEDIUM: it is suggested to have verbs as method names . can we change this method name?
```suggestion
  protected getDownloadAttachmentEndpoint(
```